prow/config.yaml

```diff
@@ -338,10 +338,9 @@ presubmits:
     rerun_command: "/test pull-kubernetes-e2e-gce-bazel"
     trigger: "(?m)^/test (all|pull-kubernetes-e2e-gce-bazel),?(\\s+|$)"
     always_run: true
-    skip_branches:
-    - release-1.4
-    - release-1.5
-    - release-1.6
+    # TODO(krzyzacy):We need to backfill to support feature branches
```

Comment:
added a todo for feature branches, https://github.com/kubernetes/test-infra/pull/4918 will resolve this, and we can also fill this in manually later if we want